rest/route/host_agent.go

```diff
@@ -322,107 +304,9 @@ func (h *hostAgentNextTask) prepareHostForAgentExit(ctx context.Context, params
 	}
 }
 
-// fixIntentHostRunningAgent handles an exceptional case in which an ephemeral
```

Comment:
Can we move these function coments over to the new existences of these functions that now live in `data/host`?

Reply:
Done.